integration-tests/utils/integration-utils.js

```diff
@@ -106,7 +106,18 @@ const clickTableCell = async (page, { tableName, columnHeader, text, isDescendan
 }
 
 const click = async (page, xpath, options) => {
-  return (await page.waitForXPath(xpath, defaultToVisibleTrue(options))).click()
+  try {
+    return (await page.waitForXPath(xpath, defaultToVisibleTrue(options))).click()
+  } catch (e) {
+    // Handle Error: Node is detached from document
+    //   at ElementHandle._scrollIntoViewIfNeeded (/mnt/ramdisk/.yarn/unplugged/puppeteer-npm-13.5.1-231e586c7a/node_modules/puppeteer/src/common/JSHandle.ts:475:22)
+    //   at processTicksAndRejections (node:internal/process/task_queues:96:5)
+    //   at ElementHandle.click (/mnt/ramdisk/.yarn/unplugged/puppeteer-npm-13.5.1-231e586c7a/node_modules/puppeteer/src/common/JSHandle.ts:617:5)
```

Comment:
This comment isn't really providing any useful information. The error message is in the `if` statement and the sample stack trace refers to Puppeteer internals. An explanation of what causes this error and why it is expected to occasionally occur would be more useful.

Reply:
Removed comment. I think the code is obvious already.